scripts/xen/gold_linux/run_alpine

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+sudo ln $(dirname $0)/alpine.cfg /etc/xen/alpine.cfg
+
+# Getting alpine linux image
+sudo mkdir /data && cd /data
+sudo wget dl-cdn.alpinelinux.org/alpine/v3.9/releases/x86_64/alpine-virt-3.9.2-x86_64.iso
```

Comment:
`wget -c` or download only if the file is absent, to avoid transferring an extra bytes